api/package.json

```diff
@@ -7,7 +7,8 @@
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "db:migrate:create": "db-migrate create -e local -m ../db/migrations --config ../db/database.json"
```

Comment:
maybe we could name it `db:migration:create`, `db:migration:up`, `db:migration:down`

Reply:
i've used the name of the tool (db-migrate), if you think it's not suitable, i can change. let me know